go.mod

```diff
@@ -115,6 +115,7 @@ require (
 	github.com/google/gnostic v0.5.7-v3refs // indirect
 	github.com/gorilla/mux v1.8.0 // indirect
 	github.com/hashicorp/go-cleanhttp v0.5.2 // indirect
+	github.com/hoisie/mustache v0.0.0-20160804235033-6375acf62c69 // indirect
```

Comment:
Lets replace usage of this pkg with the standard text/template, here's some example usage in the kubevirt/kubevirt repo, https://github.com/kubevirt/kubevirt/blob/07ea0e40919fc723778df1ad01fb74acfab260b5/tools/manifest-templator/manifest-templator.go#L227